pyproject.toml

```
@@ -23,7 +23,7 @@ maintainers = [
 	"Nate Schimmoller <nschimmo@gmail.com>",
 	"Zachary Ware <zachary.ware@gmail.com>",
 	]
-license = "LGPL 2.1"
+license = "(LGPL-2.1-only)"
```

The parens continue the existing breakage, just differently.

```suggestion
license = "LGPL-2.1-only"
```